content/en/docs/tutorials/hello-minikube.md

```diff
@@ -170,6 +170,8 @@ Kubernetes cluster. To make the `hello-node` Container accessible from outside t
 Kubernetes virtual network, you have to expose the Pod as a
 Kubernetes [*Service*](/docs/concepts/services-networking/service/).
 
+{{< warning >}} The agnhost container has a `/shell` endpoint, which is useful for debugging, but dangerous to expose to the public internet. Do not run this on an internet-facing cluster, or a production cluster. {{< /warning >}}
```

Comment:
```suggestion
{{< warning >}}
The agnhost container has a `/shell` endpoint, which is useful for
debugging, but dangerous to expose to the public internet. Do not run this on an
internet-facing cluster, or a production cluster.
{{< /warning >}}
```

Reply:
Small Nit: Wrap the lines

Reply:
By "internet", I think you meant "Internet".

Reply:
> By "internet", I think you meant "Internet".

Most places do not capitalize internet anymore. Though it [looks like it is mixed](https://en.wikipedia.org/wiki/Capitalization_of_Internet).

I don't think we've documented its usage in the style guide, but personally I have not capitalized it for at least a decode now.

Reply:
I wouldn't capitalize for “internet-facing”. I might for “The Internet”.

Reply:
@sftim / @Shubham82 - I've a question if you don't mind. If the suggestion is to wrap the lines then how come at lines 146 to 148, the same is not done?

```
   {{< note >}}
   Replace `hello-node-5f76cf6ccf-br9b5` in the `kubectl logs` command with the name of the pod from the `kubectl get pods` command output.
   {{< /note >}}
```